src/main/java/uk/gov/hmcts/divorce/document/print/model/Letter.java

```diff
@@ -1,14 +1,29 @@
 package uk.gov.hmcts.divorce.document.print.model;
 
-import lombok.AllArgsConstructor;
 import lombok.Getter;
 import uk.gov.hmcts.ccd.sdk.type.ScannedDocument;
+import uk.gov.hmcts.divorce.document.model.ConfidentialDivorceDocument;
 import uk.gov.hmcts.divorce.document.model.DivorceDocument;
 
-@AllArgsConstructor
 @Getter
 public class Letter {
-    private final DivorceDocument divorceDocument;
-    private final ScannedDocument scannedDocument;
+    private DivorceDocument divorceDocument;
```

Comment:
why did you get rid of the `final` stuff?

Reply:
as we have now added multiple constructor overloads with required fields only, so we don't have to set the optional fields as final otherwise we will have to set them as null in each constructor.